src/app/ApplicationTemplate.Tests/ApplicationTemplate.Tests.csproj

```diff
@@ -37,8 +37,8 @@
 		<PackageReference Include="Serilog.Settings.Configuration" Version="3.1.0" />
 		<PackageReference Include="Serilog.Sinks.Debug" Version="1.0.1" />
 		<PackageReference Include="Moq" Version="4.13.0" />
-		<PackageReference Include="FluentAssertions" Version="5.9.0" />
-		<PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.6.2" />
+		<PackageReference Include="FluentAssertions" Version="6.6.0" />
```

Comment:
Was this update intentional?

Reply:
see comment below

Reply:
I'm talking about the FluentAssertions update, which is not related to FluentValidation.

Reply:
Fluent packages? They are totally unrelated.

Reply:
They are not from the same people.